packages/store/src/schemas/Mixed.sol

```diff
@@ -12,131 +13,166 @@ import { EncodeArray } from "../tightcoder/EncodeArray.sol";
 import { Schema, SchemaLib } from "../Schema.sol";
 import { PackedCounter, PackedCounterLib } from "../PackedCounter.sol";
 
-// -- User defined schema --
-
 struct Mixed {
   uint32 u32;
   uint128 u128;
   uint32[] a32;
   string s;
 }
 
-// -- Autogenerated library --
-// TODO: autogenerate
-
 library Mixed_ {
   /** Get the table's schema */
-  function getSchema() internal pure returns (Schema schema) {
-    schema = SchemaLib.encode(SchemaType.UINT32, SchemaType.UINT128, SchemaType.UINT32_ARRAY, SchemaType.STRING);
+  function getSchema() internal pure returns (Schema) {
+    SchemaType[] memory _schema = new SchemaType[](4);
+    _schema[0] = SchemaType.UINT32;
+    _schema[1] = SchemaType.UINT128;
+    _schema[2] = SchemaType.UINT32_ARRAY;
+    _schema[3] = SchemaType.STRING;
+
+    return SchemaLib.encode(_schema);
   }
 
   /** Register the table's schema */
   function registerSchema(uint256 tableId) internal {
     StoreSwitch.registerSchema(tableId, getSchema());
   }
 
-  function registerSchema(uint256 tableId, IStore store) internal {
-    store.registerSchema(tableId, getSchema());
+  function registerSchema(uint256 _tableId, IStore _store) internal {
```

Comment:
ooc why the change to underscore-prefixed arguments?

Reply:
I only allow field/key names that start with `[a-z]`, so they're guaranteed to never clash with underscore-prefixed internal stuff